pathvalidate/_common.py

```diff
@@ -138,3 +138,10 @@ def normalize_platform(name: Optional[PlatformType]) -> Platform:
 
 def findall_to_str(match: List[Any]) -> str:
     return ", ".join([repr(text) for text in match])
+
+
+def truncate_str(text: str, encoding: str, max_bytes: int):
```

Comment:
Could you add a type annotation for the return value?

Reply:
Done!